pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -52,7 +52,7 @@ stdenv.mkDerivation rec {
     librsvg # for loading icons
 
     python2Env perlPackages.perl
-  ];
+  ] ++ stdenv.lib.optional (!stdenv.isDarwin) gtkspell2;
```

Comment:
Looking at ofBorg complaints, I think this check doesn't work as expected (and `stdenv.isLinux` is probably a better condition)

Reply:
Ah, I guess `isDarwin` is good enough.